tests/plugins/wysiwygarea/manual/detached.md

```diff
@@ -0,0 +1,23 @@
+@bender-tags: 4.16.0, bug, 4462, wysiwygarea
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+
+1. Open dev console
+2. Click "Toggle" button twice. To hide and show editor.
+
+	**Expected:** Editor shows with same data. Content is editable. Elementspath (at the bottom is updated).
+
+	**Unexpected:** Editor data is lost, content area is not editable.
+
+3. Write some text in editor.
+
+4. Repeat step 2.
```

Comment:
In general the `Expected/Unexpected` is not a part of a step (and can be easily omitted if tester isn't very focused 😜 ), so it's better to repeat them every time they should be checked (so after 2, 4, 6 and 8).